lxd/patches.go

```diff
@@ -907,13 +1039,13 @@ func upgradeFromStorageTypeZfs(name string, d *Daemon, defaultPoolName string, d
 			fmt.Sprintf("mountpoint=%s", newContainerMntPoint),
 			ctDataset).CombinedOutput()
 		if err != nil {
-			return fmt.Errorf("Failed to set new ZFS mountpoint: %s.", output)
+			shared.LogWarnf("Failed to set new ZFS mountpoint: %s.", output)
 		}
 
 		// Check if we need to account for snapshots for this container.
 		ctSnapshots, err := dbContainerGetSnapshots(d.db, ct)
 		if err != nil {
-			return err
+			continue
```

Comment:
There's no reason this would fail unless the DB is broken, if the DB is broken we don't want to continue, so this should be kept as fatal.